sympy/physics/vector/frame.py

```diff
@@ -127,6 +129,22 @@ def __init__(self, name, indices=None, latexs=None, variables=None):
         >>> vlatex(P.x)
         'A1'
 
+
+        Example to create multiple ReferenceFrames:
```

Comment:
This should be one space between the proceeding line.

Reply:
I'd change this to "`symbols()` can be used to create several reference frames in one step, for example:".